pkg/auth/auth.go

```diff
@@ -145,8 +172,14 @@ func Auth(adminKey, keyString string) (*SignedInUser, error) {
 		return nil, err
 	}
 
-	// add the user to the cache.
-	log.Debug("Caching validKey response for %d seconds", validTTL/time.Second)
-	cache.Set(keyString, user, validTTL)
-	return user, nil
+	for _, id := range validOrgIds {
```

Comment:
lets keep this backwards compatible.   `if len(validOrgIds) == 0 `, then consider the org to be valid.